src/test/java/se/sundsvall/partyassets/service/StatusServiceTest.java

```diff
@@ -93,7 +96,7 @@ void createReasons() {
 		final var reasons = List.of("BLOCKED_REASON_1", "BLOCKED_REASON_2");
 
 		// Act
-		service.createReasons(status, reasons);
+		service.createReasons(MUNICIPALITY_ID,status, reasons);
```

Comment:
Separate attributes with space (i.e `MUNICIPALITY_ID,status` -> `MUNICIPALITY_ID, status`. Comment goes for all occurrences in code.